client/src/Components/Login.jsx

```diff
@@ -23,42 +31,74 @@ export default function Login({ user, setUser }) {
     setLoginData({ ...loginData, [e.target.id]: e.target.value });
   };
 
-  const handleCheck = () => {
-    const token = Cookies.get('authToken');
-    const userData = Cookies.get('username');
-    console.log('user: ', user);
-    console.log('token: ', token);
-    console.log('userCookie: ', userData);
+  const goBack = () => {
+    navigate(-1);
   };
 
   return (
-    <div>
-      <form>
-        <label htmlFor="email">
-          email:
+    <div className="login-signup-container main-container">
```

Comment:
This comment applies to both here and Register.jsx to address a few small things. Semantic HTML imo should be more used here tags such as <section> <article> <aside> instead of evertying to be a div. Great use of the form tag and others but the divs, feel a little excessive. <small is well ive never seen it, but its semantic no need to remove haha.